### ISSUE TITLE
Can you push a release with the watchDir change?

### ISSUE THREAD

**Issue by yyyc514** [thgh/rollup-plugin-scss#43]
And is it `watch` or `watchDir`? Your README might be wrong.

**Comment by yyyc514** [comment]
For anyone wanting to hack it:

```js
let scssPlugin = scss({
  output: "extension/styles.css",
  prefix: `@import "./colors.scss";`,
})
let transform = scssPlugin.transform
scssPlugin.transform = function(...args) {
  this.addWatchFile('src/css');
  return transform(...args)
}

```

**Comment by angryziber** [comment]
I also was confused with `watch` option in readme...

**Comment by angryziber** [comment]
Also, it seems that `watchDir` doesn't work recursively... Array support may be needed

**Comment by thgh** [comment]
Released in v2.2.0 including array support :-)
